test/Fail/ExactSplitBerry.err

```diff
@@ -1,4 +1,5 @@
 ExactSplitBerry.agda:9,1-10,21
+warning: -W[no]CoverageNoExactSplit
```

Comment:
This case is also a bit strange.  The user turns on `--exact-split` (maybe project-wide) but then would turn off the warning?  Is there a realistic scenario for this?   Why not turning off `--exact-split` instead?
Or should we just see `--exact-split` as an alias for `-WCoverageNoExactSplit`?  
@jespercockx, what do you think?

Reply:
Hm yeah it seems like these are just two names for the same thing: `--exact-split` is a warning flag "avant la lettre"; if it were introduced today it would just be a warning (which is off by default).